releases/net471/KnownIssues/481090-WPF Touch generates NullReferenceException in ProcessInputReport.md

```diff
@@ -0,0 +1,12 @@
+## WPF Touch generates a NullReferenceException in WispLogic.ProcessInputReport
+
+### Symptoms
+A NullReferenceException is thrown and the application might crash.
```

Comment:
NullReferenceException --> `T:System.NullReferenceException`
instead of passive, can this be made active; e.g., "WPF throws a NullReferenceException, and the application..."
Also, is there an exception message that helps developers determine whether the NullReferenceException is caused by this or something else?